packages/cozy-flags/package.json

```diff
@@ -29,7 +29,6 @@
     ]
   },
   "dependencies": {
-    "detect-node": "2.0.4",
```

Comment:
weird that there is no change in a yarn.lock file?

Reply:
Eagle eye !

Packages in the lerna repository do not have `yarn.lock`s. `detect-node` is still used by other packages (cozy-stack-client and another dep), this is why it has not disappeared.

Extract from the yarn.lock.

```

3788-cozy-stack-client@^6.25.0:
3789-  version "6.25.0"
3790-  resolved "https://registry.yarnpkg.com/cozy-stack-client/-/cozy-stack-client-6.25.0.tgz#4fc50087d900be4b0976ceca3606c77703cf5f52"
3791-  integrity sha512-SY7450Yg9n9HDcPsMaNgfTcId5rXOR7hpzyCo1OUZoLINOGlG9Crd10p01aiOLSbvw8z1piKJnGyxYSSrmBOXA==
3792-  dependencies:
3793:    detect-node "2.0.4"

14023-spdy-transport@^2.0.18:
14024-  version "2.1.1"
14025-  resolved "https://registry.yarnpkg.com/spdy-transport/-/spdy-transport-2.1.1.tgz#c54815d73858aadd06ce63001e7d25fa6441623b"
14026-  integrity sha512-q7D8c148escoB3Z7ySCASadkegMmUZW8Wb/Q1u0/XBgDKMO880rLQDj8Twiew/tYi7ghemKUi/whSYOwE17f5Q==
14027-  dependencies:
14028-    debug "^2.6.8"
14029:    detect-node "^2.0.3"
```

Reply:
Thank !